tests/data/expected_jsonschemas/expected.Biospecimen.schema.json

```diff
@@ -4,24 +4,28 @@
   "description": "TBD",
   "properties": {
     "Component": {
+      "description": "TBD",
       "minLength": 1,
       "not": {
         "type": "null"
       }
     },
     "Patient ID": {
+      "description": "TBD",
```

Comment:
Not related to this PR, but why do we have: 
```
      "minLength": 1,
      "not": {
        "type": "null"
      }
```
This seems pretty redundant to me. If minLength is already 1, then the type is not null?

Reply:
I think an empty string '' doesn't count as null, so I think this means that there must be a value, AND it must must be lengeth at least 1

Reply:
Thanks, that makes sense.